docs/install.md

```diff
@@ -4,15 +4,36 @@ The following describes how to install MlAgility.
 
 ## Installing mlagility locally
 
-Install the `mlagility` package into your environment simply run
+If you want to install the `mlagility` package into your own Python environment, simply clone this repository and run
 
 ```
 pip install -e .
 ```
 
-inside the mlagility repository.
+inside the mlagility directory.
 
-> _Note_: If you are planning to use `mlagility` with Groq or Slurm please see the corresponding sections below.
+However, if you are just getting started, we recommend that you use a [miniconda](https://docs.conda.io/en/latest/miniconda.html) environment like this:
+
+```
+git clone https://github.com/groq/mlagility.git
+conda create -n mla python=3.8
+conda activate mla
+cd mlagility
+pip install -e .
+pip install -r models/requirements.txt
```

Comment:
already shown below

Reply:
I am considering the case where someone is only going to read one instruction 👀

Reply:
Done!